pythonforandroid/bootstrap.py

```diff
@@ -137,6 +137,23 @@ def name(self):
         modname = self.__class__.__module__
         return modname.split(".", 2)[-1]
 
+    def _copy_in_final_files(self):
+        if self.name == "sdl2":
+            # Get the paths for copying SDL2's java source code:
+            sdl2_recipe = Recipe.get_recipe("sdl2", self.ctx)
+            sdl2_build_dir = sdl2_recipe.get_jni_dir()
+            src_dir = join(sdl2_build_dir, "SDL", "android-project",
+                           "app", "src", "main", "java",
+                           "org", "libsdl", "app")
+            target_dir = join(self.dist_dir, 'src', 'main', 'java', 'org',
+                              'libsdl', 'app')
+
+            # Do actual copying:
+            info('Copying in SDL2 .java files from: ' + str(src_dir))
+            if not os.path.exists(target_dir):
+                os.makedirs(target_dir)
```

Comment:
We could save one line by simulating the `--parents` flag:
```suggestion
            os.makedirs(target_dir, exist_ok=True)
```

Reply:
I think I wrote it like this to maintain Python 2 compatibility. So yes, for 3.x-only shortening it like this makes sense :+1: